src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Device;
+using System.Device.I2c;
+using System.IO;
+using System.Numerics;
+using System.Text;
+using System.Threading;
+
+namespace Iot.Device.LidarLiteV3
+{
+    /// <summary>
+    /// Lidar Lite v3 is a long-range fixed position distance sensor by Garmin.
+    /// </summary>
+    public class LidarLiteV3 : IDisposable
+    {
+        /// <summary>
+        /// Default address for LidarLiteV3
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x62;
+
+        internal I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Initialize the LidarLiteV3
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device</param>
+        public LidarLiteV3(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+            Reset();            
+        }
+
+        /// <summary>
+        /// Reset FPGA, all registers return to default values
+        /// </summary>
+        public void Reset() 
+        {
+            try {
+                WriteRegister(Register.ACQ_COMMAND, 0x00);
+            } catch (System.IO.IOException)
+            {
+                // This exception is expected due to the reset.                
+            }
+        }
+
+        /// <summary>
+        /// Measure distance
+        /// </summary>
+        public ushort MeasureDistance(bool withReceiverBiasCorrection = true) 
+        {
+            if(withReceiverBiasCorrection) {
+                WriteRegister(Register.ACQ_COMMAND, 0x04);
+            } else {
+                WriteRegister(Register.ACQ_COMMAND, 0x03);
+            }
+
+            SystemStatus status = GetStatus();                
```

Comment:
what's the expected time to wait here? If less than 12ms please consider using SpinWait for waiting instead (anywhere 1-12ms Thread.Sleep is acceptable, for less I'd go with SpinWait)

Reply:
The device is busy when initially powered on for 22ms, and each measurement takes ~ 4 ms.  My thought is to add `Thread.Sleep(22)` after a PowerOn(), and use `SpinWait()` here for the 4ms.  I never used `SpinWait()` before though, how do I wait 4ms?

Reply:
We have DelayHelpers class in a repo for that although with 4ms I'm fine with Thread.Sleep (search for that for usage examples)

Reply:
Updated to use DelayHelpers.